recipe/session/sessmodels/models.go

```diff
@@ -103,22 +138,16 @@ type RegenerateAccessTokenResponse struct {
 }
 
 type TypeInput struct {
-	CookieSecure             *bool
-	CookieSameSite           *string
-	SessionExpiredStatusCode *int
-	InvalidClaimStatusCode   *int
-	CookieDomain             *string
-	AntiCsrf                 *string
-	Override                 *OverrideStruct
-	ErrorHandlers            *ErrorHandlers
-	Jwt                      *JWTInputConfig
-	GetTokenTransferMethod   func(req *http.Request, forCreateNewSession bool, userContext supertokens.UserContext) TokenTransferMethod
-}
-
-type JWTInputConfig struct {
-	Issuer                           *string
-	Enable                           bool
-	PropertyNameInAccessTokenPayload *string
+	CookieSecure                                 *bool
+	CookieSameSite                               *string
+	SessionExpiredStatusCode                     *int
+	InvalidClaimStatusCode                       *int
+	CookieDomain                                 *string
+	AntiCsrf                                     *string
+	Override                                     *OverrideStruct
+	ErrorHandlers                                *ErrorHandlers
+	GetTokenTransferMethod                       func(req *http.Request, forCreateNewSession bool, userContext supertokens.UserContext) TokenTransferMethod
+	ExposeAccessTokenToFrontendInCookieBasedAuth *bool
```

Comment:
this can just be a bool type. Since the default value of bool in golang is `false` anyway

Reply:
Fixed in https://github.com/supertokens/supertokens-golang/pull/273/commits/4a6a9339ba9b17e9383ed7b2704c1a38f36b3b3b